src/website/src/app/documentation/get-started/get-started.component.html

```diff
@@ -150,21 +150,30 @@ <h4 id="step6-update">Step 6: Update Clarity</h4>
     <p>If you want to update to a prerelease version, you can do so by running:</p>
     <clr-code-snippet
             clrLanguage="bash"
-            [clrCode]="'ng update @angular/core@next @angular/cli@next
-ng update @clr/angular@next'"
+            [clrCode]="'ng update @angular/core@next @angular/cli@next ng update @clr/angular@next'"
             [clrDisablePrism]="true"></clr-code-snippet>
     <h2 id="browserSupport">Device and Browser Support</h2>
     <p class="bump-down"><img src="assets/images/get-started/device_support.png" alt="Device and Browser Support"
                               class="hidden-sm-down img-fluid"></p>
     <ul class="hidden-md-up">
-        <li>Internet Explorer 10 &amp; 11</li>
+        <li>Internet Explorer 11</li>
         <li>MS Edge</li>
         <li>Latest versions of Chrome, Safari, and Firefox</li>
     </ul>
     <p>
         <em>* - Note that Clarity only supports the latest two versions of the listed browsers, with the exception of
             Internet Explorer.</em>
     </p>
+    <h2>Screen Reader Support</h2>
+    <p class="bump-down">
+        Clarity has built in support for two combinations of screen reader + browser combinations. This enables support
```

Comment:
The wording here is a little bit tricky I think. We do test others, like JAWS, but these are the 'best supported on' combos.

> Clarity is best supported on the following browser and screen reader combinations. While we test across a larger set of combinations, these are the ones we certify against. We support the latest version of the browser and screen reader release.

Reply:
Updated per discussions. We can revise as needed.